src/source-dpdk.c

```diff
@@ -207,15 +212,26 @@ static void DevicePreClosePMDSpecificActions(DPDKThreadVars *ptv, const char *dr
     }
 
     if (strcmp(driver_name, "net_i40e") == 0) {
-#if RTE_VERSION > RTE_VERSION_NUM(20, 0, 0, 0)
+#if RTE_VERSION >= RTE_VERSION_NUM(20, 0, 0, 0)
+        // Flush the RSS rules that have been inserted in the post start section
```

Comment:
the code here seems the same right? I would play around the `#if` to not have code duplication here. 

e.g.  something like:

```
if ( \
#if RTE_VERSION > 20
    strcmp(..., "net_i40e") == 0 \
#endif
    strcmp(... "net_mlx5") == 0 || \
    strcmp(... "net_ice") == 0 || \
    ...

```